docs/src/main/sphinx/security/jwt.rst

```diff
@@ -0,0 +1,141 @@
+==================
+JWT authentication
+==================
+
+Trino can be configured to authenticate client access using `JSON web tokens
+<https://en.wikipedia.org/wiki/JSON_Web_Token>`_. A JWT is a small, web-safe
+JSON file that contains cryptographic information similar to a certificate,
+including:
+
+-  Subject
+-  Valid time period
+-  Signature
+
+Sites such as `jwt.io <jwt.io>`_ are available to help you decode and verify a
+JWT.
+
+A JWT is designed to be passed between servers as proof of prior authentication
+in a workflow like the following:
+
+* An end user logs into a client application and requests access to a server.
```

Comment:
Maybe use a numbered list here, since this is a sequence of steps

Reply:
OK